pkg/providers/builtin/cloudsql/mysql-definition.go

```diff
@@ -184,11 +184,11 @@ func MysqlServiceDefinition() *broker.ServiceDefinition {
 			{
 				FieldName: "instance_name",
 				Type:      broker.JsonTypeString,
-				Details:   "Name of the Cloud SQL instance.",
+				Details:   "Name of the CloudSQL instance.",
 				Default:   identifierTemplate,
 				Constraints: validation.NewConstraintBuilder().
 					Pattern("^[a-z][a-z0-9-]+$").
-					MaxLength(84).
+					MaxLength(87).
```

Comment:
Just confirming this new length is correct.

Reply:
nice catch, it looks like it's not